### PR TITLE
[Hotfix] VAR-159 | Datepicker width getting too small

### PR DESCRIPTION
**Pull request by midori0507**
- Almost revert old PR.
- Revert/removal overall border styling
- Give dynamic width only with recurring controls (middle aligned)

### CODE REVIEW

File: app/shared/date-picker/_date-picker.scss [hunk 1]
```diff
@@ -17,17 +17,21 @@
 
   input {
     width: 100%;
-    border: 1px solid $input-border;
-    padding: 5px;
+
+    &:focus {
+      outline: none
+    }
```
Comment (Nizar-Rahme) [inline app/shared/date-picker/_date-picker.scss]: For accessibility's sake, it's better to keep the outline on. 
Even if it doesn't look pretty, we can play around with its color.

File: app/shared/date-picker/_date-picker.scss [hunk 2]
```diff
@@ -87,7 +91,7 @@
     background-color: $light-gray;
     border-radius: 0;
     font-size: 1.6rem;
-  
+
     &:focus {
       outline: none;
     }
```
Comment (Nizar-Rahme) [inline app/shared/date-picker/_date-picker.scss]: And maybe here we can bring it back?

File: app/shared/recurring-reservation-controls/_recurring-reservation-controls.scss [hunk 3]
```diff
@@ -6,4 +6,23 @@
...
   }
+
+  .date-picker {
+    height: $input-height-base;
+    display: flex;
+    justify-items: center;
+    flex-direction: column;
+    border: none;
+
+    input {
```
Comment (Nizar-Rahme) [inline app/shared/recurring-reservation-controls/_recurring-reservation-controls.scss]: Is there a way to attach a className instead for targeting the element's name?

File: app/shared/date-picker/_date-picker.scss (hunk 1 above)
Comment (midori0507) [reply app/shared/date-picker/_date-picker.scss]: Really ? ._. ok then

File: app/shared/date-picker/_date-picker.scss (hunk 2 above)
Comment (midori0507) [reply app/shared/date-picker/_date-picker.scss]: It's just linter work, i think they better with space..

File: app/shared/recurring-reservation-controls/_recurring-reservation-controls.scss (hunk 3 above)
Comment (midori0507) [reply app/shared/recurring-reservation-controls/_recurring-reservation-controls.scss]: Under this component there are only 1 date-picker then i dont think its necessary

File: app/shared/date-picker/_date-picker.scss (hunk 2 above)
Comment (Nizar-Rahme) [reply app/shared/date-picker/_date-picker.scss]: Yeah, I meant the outline was turned off within a previous PR, maybe we can bring it back :)

Comment (midori0507) [reply app/shared/date-picker/_date-picker.scss]: Consider it as done

**Pull request merged**